koku/api/iam/serializers.py

```diff
@@ -18,22 +18,41 @@
 """Identity and Access Serializers."""
 # disabled module-wide due to meta-programming
 # pylint: disable=too-few-public-methods
+import random
+import string
+
 from django.core.validators import validate_email
 from django.db import transaction
 from rest_framework import serializers
 from rest_framework.validators import UniqueValidator
 
+from .email import new_user_reset_email
 from .models import Customer, ResetToken, User
 
 
+def generate_password():
+    """Generate a complex password."""
+    choices = '!@#$%^&*()_+' + string.digits \
+        + string.ascii_uppercase + string.ascii_lowercase
+
+    return ''.join(random.choices(choices, k=10))
```

Comment:
NIST recently revised their password recommendations (see, for example: https://nakedsecurity.sophos.com/2016/08/18/nists-new-password-rules-what-you-need-to-know/ )

I'm pointing this out mostly to suggest that maybe we should make that `k` parameter a configurable setting. This isn't something worth blocking the PR on, though.